docs/user/auth-resources.md

```diff
@@ -0,0 +1,56 @@
+# Authentication resources
+
+The table below contains the routes, names, resource names, and verbs for flightctl API endpoints:
+
+|Route| Name| Resource| Verb |
+|-----|-----|---------|------|
+|`DELETE /api/v1/certificatesigningrequests`|`DeleteCertificateSigningRequests`|`certificatessigningrequests`|`deletecollection`|
+|`GET /api/v1/certificatesigningrequests`|`ListCertificateSigningRequests`|`certificatessigningrequests`|`list`|
+|`POST /api/v1/certificatesigningrequests`|`CreateCertificateSigningRequest`|`certificatessigningrequests`|`create`|
+|`DELETE /api/v1/certificatesigningrequests/{name}`|`DeleteCertificateSigningRequest`|`certificatessigningrequests`|`delete`|
+|`GET /api/v1/certificatesigningrequests/{name}`|`ReadCertificateSigningRequest`|`certificatessigningrequests`|`read`|
+|`PATCH /api/v1/certificatesigningrequests/{name}`|`PatchCertificateSigningRequest`|`certificatessigningrequests`|`patch`|
+|`PUT /api/v1/certificatesigningrequests/{name}`|`ReplaceCertificateSigningRequest`|`certificatessigningrequests`|`replace`|
+|`DELETE /api/v1/certificatesigningrequests/{name}/approval`|`DenyCertificateSigningRequest`|`certificatessigningrequests`|`deny`|
+|`POST /api/v1/devices`|`CreateDevice`|`devices`|`create`|
+|`GET /api/v1/devices`|`ListDevices`|`devices`|`list`|
+|`DELETE /api/v1/devices`|`DeleteDevices`|`devices`|`deletecollection`|
+|`GET /api/v1/devices/{name}`|`ReadDevice`|`devices`|`read`|
+|`PUT /api/v1/devices/{name}`|`ReplaceDevice`|`devices`|`replace`|
+|`DELETE /api/v1/devices/{name}`|`DeleteDevice`|`devices`|`delete`|
+|`GET /api/v1/devices/{name}/status`|`ReadDeviceStatus`|`devices`|`readstatus`|
+|`PUT /api/v1/devices/{name}/status`|`ReplaceDeviceStatus`|`devices`|`replacestatus`|
+|`GET /api/v1/devices/{name}/rendered`|`GetRenderedDeviceSpec`|`devices`|`getrenderedspec`|
+|`PUT /api/v1/devices/{name}/decommission`|`DecommissionDevice`|`devices`|`decomission`|
+|`POST /api/v1/enrollmentrequests`|`CreateEnrollmentRequest`|`enrollmentrequests`|`create`|
+|`GET /api/v1/enrollmentrequests`|`ListEnrollmentRequests`|`enrollmentrequests`|`list`|
+|`DELETE /api/v1/enrollmentrequests`|`DeleteEnrollmentRequests`|`enrollmentrequests`|`deletecollection`|
+|`GET /api/v1/enrollmentrequests/{name}`|`ReadEnrollmentRequest`|`enrollmentrequests`|`read`|
+|`PUT /api/v1/enrollmentrequests/{name}`|`ReplaceEnrollmentRequest`|`enrollmentrequests`|`replace`|
+|`DELETE /api/v1/enrollmentrequests/{name}`|`DeleteEnrollmentRequest`|`enrollmentrequests`|`delete`|
+|`GET /api/v1/enrollmentrequests/{name}/status`|`ReadEnrollmentRequestStatus`|`enrollmentrequests`|`readstatus`|
+|`POST /api/v1/enrollmentrequests/{name}/approval`|`ApproveEnrollmentRequest`|`enrollmentrequests`|`approve`|
```

Comment:
```suggestion
|`POST /api/v1/enrollmentrequests/{name}/approval`|`ApproveEnrollmentRequest`|`enrollmentrequests/approval`|`update`|
```

Reply:
Again looks like an issue in our API that we use POST instead of PUT...

Reply:
It is not in this PR scope

Reply:
What I meant is you correctly modelled it after our current API, but the API should be different.

Reply:
That's why I said it is not for this PR. Once the API changes, the auth model should change too.